types/hellojs/index.d.ts

```diff
@@ -146,3 +170,6 @@ declare namespace hello {
         logout?: ((callback: () => void | string) => void) | string;
     }
 }
+declare global {
+    let hello: hello.HelloJSStatic;
```

Comment:
This shouldn't be necessary given the `export as namespace` at the top.

Reply:
thanks! just pushed a new version, hopefully this one is the good one